src/phoenix/trace/__init__.py

```diff
@@ -8,3 +11,16 @@
     "DocumentEvaluations",
     "TraceEvaluations",
 ]
+
+
+@contextlib.contextmanager
+def suppress_instrumentation() -> Iterator[None]:
```

Comment:
```suggestion
def suppress_tracing() -> Iterator[None]:
```

This wording might be clearer to the user - we could just provide an alias as well